.github/workflows/push-release.yml

```diff
@@ -41,6 +54,10 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      # see note in any-pr.yml about python 3.10.11
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10.11"
```

Comment:
Could we use `pip install setuptools` like you did up here?

Reply:
Thank you, done!